source/blog/2016-08-01-ocitools-libgen.html.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Creating OCI configurations with the ocitools generate library
+author: Haiyan Meng
+date: 2016-08-01 11:11:11 UTC
+tags: OCI, ocitools
+published: true
+comments: true
+---
+
+This post will guide you through the steps of creating [OCI](https://github.com/opencontainers/runtime-spec/) configurations
+using the [ocitools generate library](https://github.com/opencontainers/ocitools/tree/master/generate).
+
+There are four steps to create an OCI configurations using the ocitools generate library.
+
+1. Import the ocitools generate libraray into your project;
+2. Create an OCI specification generator;
+3. Modify the specification by calling different methods of the specification generator;
+4. Save the specification.
+
+READMORE
```

Comment:
This is just a mark to limit the amount of the content to show by default. It automatically functions as an expansion button.